src/content/documentation/develop/request-the-right-permissions.md

```diff
@@ -124,6 +123,7 @@ The following permissions don't get alerted to users:
   - `identity`
   - `idle`
   - `menus`
+  - `nativeMessaging`
```

Comment:
This isn't in the correct place.  This is a list of permissions that are not included in permission prompts (aka silent permissions).  nativeMessaging does cause a prompt, and it can be in either the permissions list or the optional_permissions list.